### ISSUE TITLE
KeyError username

### ISSUE THREAD

**Issue by xLeevo** [Noctem/Monocle#173]
Hello, I am getting the Error "KeyError: 'username' whenever i make the accounts.csv file longer than one account. if there is only one line, it is working fine tho.
http://puu.sh/uD6Cb/af759ebbcf.png

Is there any suggestion for it please?

**Comment by Noctem** [comment]
Do you have [column headings like the example](https://github.com/Noctem/Monocle/blob/develop/accounts.example.csv)?

**Comment by xLeevo** [comment]
yes, i made it like in the example. also it is working with only one account in the file, but not with 2 for example ...?

**Comment by DavePlater** [comment]
I did not fill in the ptc column and have not had trouble so far.
You are not using any odd characters in your username or password are you, that you would need to surround the entry with double quotes?
